apps/pie-docs/src/tokensMetadata.json

```diff
@@ -1508,43 +1508,43 @@
       },
       "box-shadow-01": {
         "displayName": "Box shadow 01",
-        "description": "0,2,2,0 rgba(27,35,36,.03)<br /> 0,3,1,-2 rgba(27,35,36,.07)<br /> 0,1,5,0 rgba(27,35,36,.06)"
+        "description": "0,2,2,0 rgba(0,0,0,.03)<br /> 0,3,1,-2 rgba(0,0,0,.07)<br /> 0,1,5,0 rgba(0,0,0,.06)"
```

Comment:
Just a note (and def not for this PR), but would be great if we could hook these up to the token values as part of the next phase /cc @jamieomaguire